### PR TITLE
[TRIVIAL] Update rippled-example.cfg docs to clarify usage of ssl_cert vs ssl_chain

### PR DESCRIPTION
**Pull request by ggprod**
## High Level Overview of Change

Update rippled-example.cfg docs to clarify usage of ssl_cert vs ssl_chain

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

**Comment by ckeshava** [comment]
it's strange that the CI tests failed in the previous PR but there is no problem with this PR (the change list is the same)

LGTM 👍

**Comment by ggprod** [comment]
> it's strange that the CI tests failed in the previous PR but there is no problem with this PR (the change lis

@ckeshava would it be possible to get a formal approval of the PR so that it could be merged?

**Review by ckeshava — APPROVED** [review]

File: cfg/rippled-example.cfg [hunk 1]
```diff
@@ -283,12 +283,14 @@
 #       ssl_cert
 #
 #           Specifies the path to the SSL certificate file in PEM format.
-#           This is not needed if the chain includes it.
+#           This is not needed if the chain includes it. Use ssl_chain if
+#           your certificate include one or more intermediates.
```
Comment (intelliot) [inline cfg/rippled-example.cfg:287]:
```suggestion
#           your certificate includes one or more intermediates.
```

**Comment by intelliot** [comment]
@manojsdoshi I think this would be ok to consider for 1.12.0-rc2 because it is only a trivial comment - no code change

**Pull request merged**